### PR TITLE
make the CSS for the loading animation inline to avoid blank screen

### LINKED ISSUE #4316: Optimize loading
Hi there! 
I suggest the creation of a pre-load page to minimize loading time. 
The current blank screen loading time is 1.23 seconds, because the app is downloading all css and js files before showing the splash screen. See attached network log screenshot.
screenshot from 2016-09-15 12-35-53

### PR DESCRIPTION
**Pull request by engelgabriel**
@RocketChat/core 

Closes #4316

### CODE REVIEW

File: packages/rocketchat-ui-master/server/inject.js [hunk 1]
```diff
@@ -1,14 +1,68 @@
...
+<div id="initial-page-loading" class="page-loading">
+	<div class="loading">
+		<div class="bounce1"></div>
+		<div class="bounce2"></div>
+		<div class="bounce3"></div>
+	</div>
+</div>`);
+
+
+RocketChat.settings.get('theme-color-primary-background-color', function(key, value) {
```
Comment (geekgonecrazy) [inline packages/rocketchat-ui-master/server/inject.js]: Will this introduce delay in showing background color by waiting until it calls?   If so maybe a define the default above.. then Inject new style when / if it gets a setting?

File: packages/rocketchat-ui-master/server/inject.js [hunk 2]
```diff
@@ -1,14 +1,68 @@
...
+	if (value) {
+		Inject.rawBody('theme-color-primary-background-color', `<style>body { background-color: ${value};}</style>`);
+	} else {
+		Inject.rawBody('theme-color-primary-background-color', '<style>body { background-color: #04436a;}</style>');
+	}
+});
+
+RocketChat.settings.get('theme-color-tertiary-font-color', function(key, value) {
```
Comment (geekgonecrazy) [inline packages/rocketchat-ui-master/server/inject.js]: Same wonder / concern with this one.

File: packages/rocketchat-ui-master/server/inject.js (hunk 1 above)
Comment (engelgabriel) [inline packages/rocketchat-ui-master/server/inject.js]: No, this runs only once on the server side.

File: packages/rocketchat-ui-master/server/inject.js (hunk 2 above)
Comment (engelgabriel) [inline packages/rocketchat-ui-master/server/inject.js]: No, this runs only once on the server side.

**Pull request merged**